### PR TITLE
config.yml could use some changes (Essentials)

### PR DESCRIPTION
**Pull request by xItsNickHD**
Some changes I personally believe should be changed. You don't have to approve all of it, edit the bits you think should be different.

### CODE REVIEW

File: Essentials/config.yml [hunk 1]
```diff
@@ -264,34 +265,7 @@ skip-used-one-time-kits-from-kit-list: true
...
-    delay: 6000
-    items:
-      - 387 1 title:&4Book_&9o_&6Colors author:KHobbits lore:Ingame_color_codes book:Colors
-  firework:
-    delay: 6000
-    items:
-      - 401 1 name:Angry_Creeper color:red fade:green type:creeper power:1
-      - 401 1 name:Starry_Night color:yellow,orange fade:blue type:star effect:trail,twinkle power:1
-      - 401 2 name:Solar_Wind color:yellow,orange fade:red shape:large effect:twinkle color:yellow,orange fade:red shape:ball effect:trail color:red,purple fade:pink shape:star effect:trail power:1
+# Removed kits because they're useless.
```
Comment (imAero) [inline Essentials/config.yml]: I think you should just leave the kits the way they are. If people don't like them, they can use the command **/deletekit [kit]** which I personally added to Essentials as we're probably going to end up removing KitAdder.

**Comment by imAero** [comment]
@xItsNickHD are you there?

**Comment by imAero** [comment]
@xItsNickHD last call.

**Comment by xItsNickHD** [comment]
@imAero 

My apologies, I've been extremely busy this week and unable to reach the internet. However, I've updated my commit.

Kind Regards,
Retrical.

**Pull request merged**